core/src/lib.rs

```diff
@@ -14,6 +15,10 @@ pub use bytes::Bytes;
 pub use event_id::EventId;
 pub use interest::{Interest, PeerId};
 pub use jwk::Jwk;
+pub use key::{
```

Comment:
do we really need all of these as core primitives? it seems nicer to make APIs around bytes or DIDs or SigningKeys and have the anchor crate read things from strings and directories and then convert them into whatever format we need to construct things

Reply:
talked about putting these behind a `NodeId` struct (or similar) so they have a home on something that is `core`

Reply:
https://linear.app/3boxlabs/issue/AES-342/implement-nodeid-to-encapsulate-node-private-key-utilities